src/tests/ftest/util/host_utils.py

```diff
@@ -0,0 +1,241 @@
+"""
+(C) Copyright 2018-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+from socket import gethostname
+
+from ClusterShell.NodeSet import NodeSet
+
+from slurm_utils import get_partition_hosts, get_reservation_hosts, SlurmFailed
```

Comment:
(pylint-no-name-in-module) No name 'get_partition_hosts' in module 'slurm_utils'
(pylint-no-name-in-module) No name 'get_reservation_hosts' in module 'slurm_utils'